index.html

```diff
@@ -66,6 +66,10 @@
       integrity="sha384-xOolHFLEh07PJGoPkLv1IbcEPTNtaed2xpHsD9ESMhqIYd0nLMwNLD69Npy4HI+N"
       rel="stylesheet"
     />
+    <!-- Adding the Bootstrap 5 Feature
```

Comment:
It is recommended to keep PRs minimal, one feature per PR.  But I don't care here.  So let's uncomment the Bootstrap v5 lines and delete the Bootstrap v4 lines.